src/test/java/com/hocs/test/glue/security/SecurityStepDef.java

```diff
@@ -330,5 +330,88 @@ public String monthOfDate(String date) {
     public String yearOfDate(String date) {
         return date.split("/")[2];
     }
+
+    @And("I create a DCU case for security testing")
+    public void iCreateADCUCaseForSecurityTesting() {
+        findBylinktext("Create Single Case").click();
+        waitABit(500);
+        findById("case-type-2").click(); // foi
```

Comment:
Could we change this identifier as 'place in list' ID for this list seems brittle to me as we might add more in future, maybe use the value attribute instead?